homeassistant/components/auth/login_flow.py

```diff
@@ -297,7 +297,7 @@ async def get(self, request: web.Request) -> web.Response:
         vol.Schema(
             {
                 vol.Required("client_id"): str,
-                vol.Required("handler"): vol.Any(str, list),
+                vol.Required("handler"): vol.All(list[str], vol.Length(2, 2)),
```

Comment:
```suggestion
                vol.Required("handler"): tuple[str, str],
```
Does this work? If it does, the `tuple` call below might even be redundant.

Reply:
No, it doesn't work. Also, what I tried to do does not work correctly; the correct voluptuous syntax to require a list of `str` is `[str]`, not `list[str]`

This works though:`
```
vol.Schema(vol.All([str], vol.Length(2, 2), vol.Coerce(tuple)))
```